industrial_ci/src/docker.sh

```diff
@@ -231,8 +231,8 @@ RUN apt-get update -qq \
     && apt-get -qq install --no-install-recommends -y apt-utils gnupg wget ca-certificates lsb-release
 
 RUN echo "deb ${ROS_REPOSITORY_PATH} \$(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list
-RUN apt-key adv --keyserver "${APTKEY_STORE_SKS}" --recv-key "${HASHKEY_SKS}" \
-    || { wget "${APTKEY_STORE_HTTPS}" -O - | apt-key add -; }
+RUN  { [ -n "${APTKEY_STORE_HTTPS}" ] && wget "${APTKEY_STORE_HTTPS}" -O - | apt-key add -; } \
+    || apt-key adv --keyserver "${APTKEY_STORE_SKS}" --recv-key "${HASHKEY_SKS}"
```

Comment:
If I understand this correctly, key will be downloaded with `wget` as the first option if `APTKEY_STORE_HTTPS` is provided. While before, the `wget` method was only used as a fallback option in case `hkp` failed.

I know I've had issues with `hkp` in the past, but I'm not sure whether these were related to our corporate network or whether these just happen randomly to everybody. Shouldn't a fallback `wget` attempt with a default URL be kept, just in case?

Reply:
> Shouldn't a fallback wget attempt with a default URL be kept, just in case?

If you have a default URL for the snapshots repo, I can adjust it accordingly.
BTW, the new behavior is the same as in #361.

Reply:
> If you have a default URL for the snapshots repo, I can adjust it accordingly.

I don't.

> BTW, the new behavior is the same as in #361.

Didn't notice that. It makes sense to keep it simple. I guess `APTKEY_STORE_HTTPS` usage should be added to the #361 docs at some point, so anybody having issues with hkp can know it exists and go figure out the right URL for them.

Reply:
I have implemented a retry mechanism